core/amber/src/main/scala/edu/uci/ics/texera/web/resource/CollaborationResource.java

```diff
@@ -0,0 +1,42 @@
+package edu.uci.ics.texera.web.resource;
+
+import edu.uci.ics.texera.web.ServletAwareConfigurator;
+
+import javax.websocket.*;
+import javax.websocket.server.ServerEndpoint;
+import java.util.HashMap;
+
+
+@ServerEndpoint(
+        value = "/collaboration",
+        configurator = ServletAwareConfigurator.class
+)
+public class CollaborationResource {
```

Comment:
and another concern is do we want to use a separate web socket to do this editing events? or combine it with the current web socket `/wsapi/workflow-websocket`?

Reply:
Ditto.